app/models/Subscription.scala

```diff
@@ -106,4 +119,8 @@ object Subscription {
       .on('subscriber_id -> subscriberId, 'topic_id -> topicId, 'action -> action, 'created -> created, 'updated -> updated, 'cancelled -> cancelled, 'earliest -> earliest, 'latest -> latest, 'active -> true).executeInsert()
     }
   }
+
+  def make(subscriberId: Int, topicId: Int, action: String, earliest: Date, latest: Date) = {
```

Comment:
I noticed during some of my Unit testing that not all models had `make`. My preference is to include it if for no other reason than it makes some of the testing easier :)